docs/notebooks/Common_Gotchas_in_JAX.md

```diff
@@ -943,7 +943,7 @@ FloatingPointError                        Traceback (most recent call last)
 
 When this code sees a nan in the output of an `@jit` function, it calls into the de-optimized code, so we still get a clear stack trace. And we can run a post-mortem debugger with `%debug` to inspect all the values to figure out the error.
 
-⚠️ You shouldn't have the NaN-checker on if you're not debugging, as it can introduce lots of device-host round-trips and performance regressions!
+⚠️ You shouldn't have the NaN-checker on if you're not debugging, as it can introduce lots of device-host round-trips and performance regressions! NaN-checker does not yet work for `@pmap` function.
```

Comment:
Maybe we can move this to a separate line and mention `vmap`, like
```suggestion
⚠️ You shouldn't have the NaN-checker on if you're not debugging, as it can introduce lots of device-host round-trips and performance regressions!

⚠️ The NaN-checker doesn't work with `pmap`. To debug nans in `pmap` code, one thing to try is replacing `pmap` with `vmap`.
```

Reply:
good idea. added